wp-content/themes/pub/wporg-learn-2024/inc/block-config.php

```diff
@@ -557,3 +559,74 @@ function inject_other_filters( $key ) {
 		printf( '<input type="hidden" name="s" value="%s" />', esc_attr( $wp_query->query['s'] ) );
 	}
 }
+
+/**
+ * Modifies the courses query for the My Courses page, by adding courses to be excluded.
+ *
+ * @param array $parsed_block The parsed block data.
+ * @return array The modified block data.
+ */
+function filter_student_course_list( $parsed_block ) {
+	if (
+		'core/query' !== $parsed_block['blockName'] ||
+		'course' !== ( $parsed_block['attrs']['query']['postType'] ?? '' ) ||
+		! isset( $parsed_block['attrs']['queryId'] ) ||
```

Comment:
This condition isn't relevant anymore, as we don't use it

Reply:
d9376a9